Search/Adapter/ZendLuceneAdapter.php

```diff
@@ -185,7 +185,9 @@ public function search(SearchQuery $searchQuery)
 
             $searcher->addIndex($this->getIndex($indexPath, false));
         }
+        if ($searchQuery->getSort() !== null) {
 
+        }
```

Comment:
Then I would leave it and remove this empty IF block.

On 20 November 2015 11:20:09 CET, Stefan Moder <notifications@github.com> wrote:
>>  
>> +        }
>
>Currently there is a "Multisearcher" used - this "Multisearcher" does
>not support sorting.
>
>Any clue how this could be done quickly?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/massiveart/MassiveSearchBundle/pull/67/files#r45453975

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.